src/Storages/Streaming/StorageRandom.h

```diff
@@ -13,7 +14,8 @@ namespace DB
 class ASTStorage;
 
 #define STORAGE_RANDOM_RELATED_SETTINGS(M) \
-    M(UInt64, eps, 1000, "Limits how many rows of data can be generated per window by random storages total streams", 0)
+    M(UInt64, eps, 1000, "Limits how many rows of data can be generated per second by random storages per stream (each query will generate $num_streams source by default). 0 means no maximum value", 0) \
+    M(UInt64, interval_time, 100, "the data generating interval, unit ms", 0)
```

Comment:
When you have `eps`, why you need `interval` ?

Reply:
when you set EPS = 2000, it means generating 2000 pieces of data per second. when you set interval_time = 200ms, it means dividing one second into equal parts of 200ms and every interval will generate 400 pieces of data.The reason for this setting is to generate data evenly within one second.